src/requests.css

```diff
@@ -11,5 +11,13 @@
 
 .requestForm {
   height: 100%;
+  width: 100%;
   overflow: auto;
 }
+
+.footerContent {
+  max-width: 50em;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
```

Comment:
Is the width really needed? Since the block is flex now, I think the container will fit 100% just because of that.